### PR TITLE
[bug 1344680] add Mozilla India distribution

### PR DESCRIPTION
**Pull request by TannerFilip**

### CODE REVIEW

**Comment by flamingspaz** [comment]
retest this please

File: distributions.tf [hunk 1]
```diff
@@ -30,3 +30,15 @@ module "badges-mozilla-org" {
...
+  source              = "git://github.com/mozilla/partinfra-terraform-cloudfrontssl.git"
+
+  origin_domain_path  = "devs.mozillaindia.org"
+  origin_path         = "/homepage"
+  origin_id           = "gh-pages-dev-mozillaindia-org"
+  alias               = "mozillaindia.org"
+  comment             = "Bug 1344680"
+  acm_certificate_arn = "${lookup(var.ssl_certificates, "community-sites-elb-${var.aws_region}")}"
+  }
+}
```
Comment (flamingspaz) [inline distributions.tf]: Remove this bracket, it doesn't match to anything

**Comment by TannerFilip** [comment]
updated

File: distributions.tf [hunk 2]
```diff
@@ -30,3 +30,14 @@ module "badges-mozilla-org" {
   comment             = "Bug 1230266"
   acm_certificate_arn = "${lookup(var.ssl_certificates, "mozilla-org-elb-${var.aws_region}")}"
 }
+
+module "mozillaindia-org" {
+  source              = "git://github.com/mozilla/partinfra-terraform-cloudfrontssl.git"
+
+  origin_domain_path  = "devs.mozillaindia.org"
```
Comment (flamingspaz) [inline distributions.tf]: should be `origin_domain_name`

**Comment by flamingspaz** [comment]
https://www.youtube.com/watch?v=u-u7CSxbe6w

**Pull request merged**